src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html

```diff
@@ -0,0 +1,225 @@
+<div>
+  <h2 class="color-blue" *ngIf="searchedTerms == -1">Students</h2>
+  <!-- Section -->
+  <h3 *ngIf="searchedTerms == -1">Student Records</h3>
```

Comment:
Can these headers be shown if the search term matches its content?

Reply:
Yup they can! Have changed it accordingly